boot/freeldr/freeldr/ui/directui.c

```diff
@@ -241,7 +241,7 @@ UiDrawProgressBar(IN ULONG Left,
     if (Position > Range) Position = Range;
 
     /* Draw the "Loading..." text */
-    UiDrawCenteredText(Left + 2, Top + 1, Right - 2, Top + 1, ProgressText, ATTR(7, 0));
+    UiDrawCenteredText(Left, Top + 1, Right, Top + 1, ProgressText, ATTR(7, 0));
```

Comment:
This solution looks more like a hacky workaround for me.

You can replace the string "Loading RamDisk..." [here](https://git.reactos.org/?p=reactos.git;a=blob;f=boot/freeldr/freeldr/disk/ramdisk.c;hb=911031e840e06f4097a6a0d3d9e3ba5dad45d1e9#l125) with a one or two character string, and then test whether your code handles it well. :thinking:

Reply:
@feel-the-dz3n : any thoughts about that?